landoapi/models/landing_job.py

```diff
@@ -246,6 +253,7 @@ def serialize(self):
                 {"revision_id": "D{}".format(r), "diff_id": self.revision_to_diff_id[r]}
                 for r in self.revision_order
             ],
+            "error_breakdown": self.error_breakdown,
```

Comment:
Does this need to be added to the Swagger definition? (should probably review the consistency of swagger at some point, against the actual DB)